core/rend/texconv.cpp

```diff
@@ -148,9 +148,9 @@ void palette_update()
 		}
 	}
 	for (std::size_t i = 0; i < std::size(pal_hash_16); i++)
-		pal_hash_16[i] = XXH32(&PALETTE_RAM[i << 4], 16 * 4, 7);
+		pal_hash_16[i] = XXH3_64bits(&PALETTE_RAM[i << 4], 16 * 4);
 	for (std::size_t i = 0; i < std::size(pal_hash_256); i++)
-		pal_hash_256[i] = XXH32(&PALETTE_RAM[i << 8], 256 * 4, 7);
+		pal_hash_256[i] = XXH3_64bits(&PALETTE_RAM[i << 8], 256 * 4);
```

Comment:
The palette hash is used in the calculation of texture hashes, so we don't want it to change either.